phoenicis-apps/src/main/java/com/playonlinux/apps/dto/ApplicationDTO.java

```diff
@@ -70,14 +71,18 @@ public String getDescription() {
         return scripts;
     }
 
+    public static Comparator<ApplicationDTO> nameComparator() {
```

Comment:
Just an idea. If it is more effort and introduced complexity, leave it as it is.